workshop1/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.16-bullseye as builder
```

Comment:
do we want to define the linux version of the docker? is it possible or is it affected by the host version?

(to avoid having problems with bcc on ubuntu 20_04 with wrong llvm version)

Reply:
i'm using debian 11 (a.k.a bullseye)
the you wont have installation problems as the docker contains a good version of the BCC

furthermore, I've already compiled the docker and uploaded to our public repo